### PR TITLE
E11S: Timeline updates

### PR DESCRIPTION
**Pull request by JLGarber**
I had a look through some of quisquous's enrage logs from this week and made some changes. The biggest addition is of course the cycle blocks and enrage at the end. Those ones I'm 100% confident on.

A *lot* of the Bound of Faith stuff seems a bit inscrutable. It got to the point where I felt like I was just throwing random extra IDs in to make things work. As it stands, this version passed every iteration of `test_timeline` I threw at it, which was about 10 different pulls. A couple of them had 0.5-1s drift on the "move to center" timing during the cycles, and the exact timing on Sinsight/Sinsmite/Sinmoke seems to just vary no matter where it is. Other than that, the timings seemed reasonably rigid.

Suggestions welcome. I'm not at this encounter yet, so everything I've done here is from viewing streams and *lots* of re-running `test`.

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e11s.txt [hunk 1]
```diff
@@ -52,35 +58,36 @@ hideall "--sync--"
...
-169.4 "Sinsmoke" sync /:Fatebreaker's image:5681:/
-169.4 "Sinsmite" sync /:Fatebreaker's image:5684:/
-169.8 "Bow Shock" sync /:Fatebreaker's image:5685:/
-174.7 "Burnout" sync /:Fatebreaker's image:567C:/
-178.0 "Blastburn" sync /:Fatebreaker's image:567A:/
+169.4 "Sinsmoke" sync /:Fatebreaker's Image:5681:/
+169.4 "Sinsmite" sync /:Fatebreaker's Image:5684:/
+169.8 "Bow Shock" sync /:Fatebreaker's Image:5685:/
+174.7 "Burnout" sync /:Fatebreaker's Image:567C:/
+178.0 "Blastburn" sync /:Fatebreaker's Image:567A:/
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e11s.txt:80]: Can you add the burnt strikes here from the adds? This is one thing fflogs didn't have, and similarly below?

File: ui/raidboss/data/05-shb/raid/e11s.txt [hunk 2]
```diff
@@ -89,49 +96,142 @@ hideall "--sync--"
...
+706.2 "Burnout" sync /:Fatebreaker:5696:/
+709.6 "Floating Fetters" sync /:Fatebreaker:5920:/
+711.6 "Solemn Charge" sync /:Fatebreaker:5697:/
+713.1 "Sinsmite" sync /:Fatebreaker:5698:/
+713.2 "Bow Shock" sync /:Fatebreaker:5699:/
+725.7 "Burnished Glory" sync /:Fatebreaker:56A4:/
+
+728.2 "--center--" sync /:Fatebreaker:5908:/ window 30,30
+735.0 "Burnished Glory?"
+737.2 "--Cycle Of Faith--" sync /:Fatebreaker:568A:/ jump 900.0
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e11s.txt:178]:
```suggestion
737.2 "Cycle Of Faith" sync /:Fatebreaker:568A:/ jump 900.0
```

File: ui/raidboss/data/05-shb/raid/e11s.txt [hunk 3]
```diff
@@ -89,49 +96,142 @@ hideall "--sync--"
...
+937.2 "--sync--" sync /:Fatebreaker:569A:/ jump 800
+937.3 "Elemental Break"
+939.8 "Sinsight/Sinsmite/Sinsmoke"
+941.7 "Burnt Strike?"
+942.0 "Blastburn?"
+946.7 "Shining Blade?"
+946.7 "Floating Fetters"
+
+
+992.3 "Burnished Glory" sync /:5529:Fatebreaker/ window 1000,5 duration 9.7
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e11s.txt:236]:
```suggestion
992.3 "Burnished Glory" sync /:5529:Fatebreaker/ window 1000,0

File: ui/raidboss/data/05-shb/raid/e11s.txt (hunk 2 above)
Comment (JLGarber) [reply ui/raidboss/data/05-shb/raid/e11s.txt:178]: Whoops! Copypasta leftovers from rearranging the order of Cycle blocks!

**Comment by quisquous** [comment]
Thank you!!

**Pull request merged**